github.js

```diff
@@ -724,6 +724,24 @@
           }
           _request("GET", url, null, cb);
       };
+
+      // starring
+      this.isStarred = function(owner, repo, cb) {
+        _request("GET", "/user/starred/" + owner + '/' + repo, null, function(err,res) {
```

Comment:
You can have `cb` as the last parameter so you can avoid creating an anonymous function that just invoke the callback.